src/llmcompressor/modifiers/quantization/gptq/gptq_quantize.py

```diff
@@ -161,11 +162,12 @@ def quantize_weight(
         H = torch.linalg.cholesky(H, upper=True)
         Hinv = H
     except torch._C._LinAlgError:
-        raise torch._C._LinAlgError(
+        logger.warning(
```

Comment:
wondering if we should add a check for how many times this happens? a user may see it pass with warnings when none of the layers are successfully modified?

Reply:
I'm hoping a user will notice if their console has multiple warnings, I don't think that case justifies the added logic complexity